pkg/asset/cluster/tfvars.go

```diff
@@ -338,16 +339,29 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		}
 		preexistingnetwork := installConfig.Config.GCP.Network != ""
 
-		imageRaw, err := rhcospkg.GCPRaw(ctx, installConfig.Config.ControlPlane.Architecture)
+		archName := coreosarch.RpmArch(fmt.Sprintf("%s", installConfig.Config.ControlPlane.Architecture))
+		st, err := rhcospkg.FetchCoreOSBuild(ctx)
 		if err != nil {
-			return errors.Wrap(err, "failed to find Raw GCP image URL")
+			return err
```

Comment:
Wrap errors with meaningful messages.

Reply:
OK rather than doing this in all the callers I'm going to add some more error wrapping/prefixing in the functions; that OK by you?  I tried to also do that in stream-metadata-go, see e.g.

https://github.com/coreos/stream-metadata-go/blob/7c4d5d7d95b51e9fd2043375ee4b727f68c8f96a/stream/stream_utils.go#L15

Reply:
If there is enough meaningful context to track down the error, that I am satisfied, regardless of where that context is added.